app/src/main/java/jp/developer/retia/frozenword/ui/habbits/HabbitsScreen.kt

```diff
@@ -57,22 +78,27 @@ fun Habbits(
             .wrapContentHeight()
     ) {
         items(habbits) { habbit ->
-            Habbit(habbit)
+            HabbitCard(habbit)
         }
     }
 }
 
 @Composable
-fun Habbit(
+fun HabbitCard(
     habbit: HabbitAndLog
 ) {
-    Habbit(habbit.habbit.trigger, habbit.habbit.title)
+    HabbitCard(
+        title = habbit.habbit.title,
+        simpleHabbitTitle = habbit.habbit.simpleHabbitTitle,
+        trigger = habbit.habbit.trigger
+    )
 }
 
 @Composable
-fun Habbit(
-    trigger: String,
+fun HabbitCard(
     title: String,
+    simpleHabbitTitle: String,
+    trigger: String? = null,
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/verno3632/FrozenWord/blob/f68c5d468215e4ac81eb6e63df27afc4be94b645/app/src/main/java/jp/developer/retia/frozenword/ui/habbits/HabbitsScreen.kt#L101"></span>Modifier parameter should be the first optional parameter</td>
    </tr>
  </tbody>
</table>